.config/nextest.toml

```diff
@@ -7,6 +7,8 @@ failure-output = "immediate-final"
 status-level = "slow"
 # Add retires for flaky tests
 retries = { backoff = "exponential", count = 3, delay = "5s" }
+# Limit threads to a "large" gha container max
+test-threads = 22
```

Comment:
❓ are these (and ci.yml) thread call outs needed?
> Default: number of logical CPUs

Main concern is if we move to different machines we may try to spin up 22 threads when there's only 12 or similar.

Reply:
The problem I'm trying to solve here is that the containers are limited to 22 cores, but cargo/nexttest has no way of knowing that since the system still reports the 48 cores the host has.

Reply:
2 large containers can be scheduled onto the same 48 core node.  If two builds/tests are running at the same time, both runs will be competing for resources thinking they have full access to all 48 cores.

What I'm hoping for is that builds and tests are actually faster and more reliable if we don't over load the nodes and force them into extra context switching.

Reply:
Ok doesn't look like it made the difference I hoped,  I will revert when I submit the 1.78.0-nightly upgrade.